dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -966,6 +968,75 @@ def lu_factor(a, overwrite_a=False, check_finite=True):
     return dpnp_lu_factor(a, overwrite_a=overwrite_a, check_finite=check_finite)
 
 
+def lu_solve(lu_and_piv, b, trans=0, overwrite_b=False, check_finite=True):
+    """
+    Solve an equation system, a x = b, given the LU factorization of `a`
+
+    For full documentation refer to :obj:`scipy.linalg.lu_solve`.
+
+    Parameters
+    ----------
+    (lu, piv) : {tuple of dpnp.ndarrays or usm_ndarrays}
```

Comment:
Now it is rendered in a wrong way in the documentation
```suggestion
    lu, piv : {tuple of dpnp.ndarrays or usm_ndarrays}
```